src/azul/chalice.py

```diff
@@ -176,12 +176,22 @@ def spec(self) -> JSON:
                 tag for tag in self._specs.get('tags', [])
                 if tag['name'] in used_tags
             ],
-            'servers': [{'url': self.self_url('/')}]
+            'servers': [{'url': str(self.base_url().set(path='/'))}]
```

Comment:
I would expect the value returned by `base_url` to already include a slash at the end. Is that not the case?

Reply:
This is not the case in the `base_url` method. I believe that the usages of `base_url` add in a slash at the beginning of the appended path. I am unsure if that would cause errors.